ui/src/components/LocationFieldset.js

```diff
@@ -58,7 +58,7 @@ class LocationFieldset extends Component {
       haveStartedLoadingMapScript: false,
       haveFinishedLoadingMapScript: false,
     };
-    this.mapsApiDisabled = ENV.maps_api_key == null || ENV.maps_api_key == '';
+    this.mapsApiEnabled = ENV.maps_api_key;
```

Comment:
I think it would be worth coercing to a boolean here (for clarity/to prevent people from doing weird things with this field in the future)

Reply:
Done.